test/es-module/test-typescript-commonjs.mjs

```diff
@@ -59,15 +59,28 @@ test('require a .ts file with implicit extension fails', async () => {
 });
 
 test('expect failure of an .mts file with CommonJS syntax', async () => {
-  const result = await spawnPromisified(process.execPath, [
-    fixtures.path('typescript/cts/test-cts-but-module-syntax.cts'),
-  ]);
+  const testFilePath = fixtures.path('typescript/cts/test-cts-but-module-syntax.cts');
+  const result = await spawnPromisified(process.execPath, [testFilePath]);
 
-  strictEqual(result.stdout, '');
-  match(result.stderr, /To load an ES module, set "type": "module" in the package\.json or use the \.mjs extension\./);
-  strictEqual(result.code, 1);
+  assert.strictEqual(result.stdout, '');
```

Comment:
nit: can you put the assertion between the stderr and code assertion? stderr is more likely to contain useful debug info if the test fails

Reply:
Sure, I'll update it right away

Reply:
Did you decide not to take this suggestion? If so that's fine, but it would useful to comment or thumbs down it otherwise it's hard to tell whether you simply forgot about it